app/javascript/react/components/BrandShowContainer.js

```diff
@@ -29,19 +32,44 @@ const BrandShowContainer = (props) => {
       getPizzas()
   }, [])
 
+
   const pizzaTiles = brand.pizzas.map( (pizza) => {
     return(
       <BrandShowTile name={pizza.product_name} id={pizza.id} key={pizza.id} />
     )
   })
-  
+
+  const addPizza = async (formPayload) => {
+    const response = await fetch("/api/v1/pizzas/", {
+      credentials: "same-origin",
+      method: "POST",
+      headers: {
+        "Accept": "application/json",
+        "Content-Type": "application/json"
+      },
+      body: JSON.stringify(formPayload),
+    })
+    const parsedNewPizza = await response.json()
+    if(parsedNewPizza.errors[0] === 'Pizza added succesfully.') {
+      setBrand({
+        ...brand,
+        pizzas: parsedNewPizza.pizzas
+      })
+    } else {
+      setErrors(parsedNewPizza.errors)
+    }
+  }
+    
   return(
-    <div className='grid-x grid-margin align_spaced pizzaContainer'>
-      <div className='cell small-11 text-center'>
-        <h2>{brand.name}</h2>
+    <>
+      <div className='grid-x grid-margin-x align-spaced pizzaContainer'>
```

Comment:
looks like this `div` is wrapping around all the contents here, so we don't really need the fragment